api/src/main/java/com/codeforcommunity/dto/school/SchoolListResponse.java

```diff
@@ -5,18 +5,18 @@
 public class SchoolListResponse {
 
   private int count;
-  private List<School> schools;
+  private List<SchoolSummary> schools;
 
-  public SchoolListResponse(List<School> schools, int count) {
+  public SchoolListResponse(List<SchoolSummary> schools, int count) {
     this.schools = schools;
```

Comment:
Same as the previous comment. Would this be fine or should we set this.schools equal to a copy of the past list of school summaries?